platform/wisdomctl/wisdomctl.py

```diff
@@ -0,0 +1,31 @@
+from ocifs import OCIFileSystem
+
+import click
+
+@click.group()
+def cli(): pass
+
+REMOTE = click.argument("remote")
+
+MB = 1024 ** 2
+def slowCopy(src, dest):
+    while True:
+        hunk = src.read(MB)
+        if not hunk: break
+        dest.write(hunk)
+
+@cli.command()
+@REMOTE
+@click.argument("local", type=click.File("wb"))
```

Comment:
this looks fine, although perhaps LAZY loading would be preferrable in the future.

https://click.palletsprojects.com/en/8.1.x/arguments/#file-opening-safety